scripts/dumpRobloxTypes.py

```diff
@@ -154,6 +155,7 @@
         "function FindFirstChild(self, name: string, recursive: boolean?): Instance?",
         "function FindFirstDescendant(self, name: string): Instance?",
         "function GetActor(self): Actor?",
+        "WaitForChild: ((self, name: string) -> Instance) & ((self, name: string, timeout: number) -> Instance?)",
```

Comment:
The recommended way that I would do this is by using the class function syntax like the rest of them. If you define more than one with the same name, Luau automatically converts it into an overloaded function
```suggestion
        "function WaitForChild(self, name: string): Instance",
        "function WaitForChild(self, name: string, timeout: number): Instance?"
```